packages/patternfly-4/react-core/src/experimental/components/OverflowMenu/OverflowMenuContent.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/OverflowMenu/overflow-menu';
+
+export interface OverflowMenuContentProps extends React.HTMLProps<HTMLDivElement> {
+  /** Any elements that can be rendered in the menu */
+  children?: any;
+  /** Additional classes added to the OverflowMenuContent */
+  className?: string;
+  /** Indicates breakpoint at which to switch between horizontal menu and vertical dropdown */
+  breakpoint?: 'Md' | 'Lg' | 'Xl';
+}
+
+export interface OverflowMenuContentState extends React.HTMLProps<HTMLDivElement> {
+  isHidden: boolean;
+}
+
+export class OverflowMenuContent extends React.Component<OverflowMenuContentProps, OverflowMenuContentState> {
+  constructor(props: OverflowMenuContentProps) {
+    super(props);
+    this.state = {
+      isHidden: false
+    }
+  }
+
+  componentDidMount() {
+    this.handleResize();
+    window.addEventListener('resize', this.handleResize);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.handleResize);
+  }
+
+  handleResize = () => {
+    const { breakpoint } = this.props;
+    const breakpointWidth = (breakpoint === 'Md')
+      ? 768
```

Comment:
I think these variables are exported somewhere in react-tokens so we don't have to use the hardcoded values.

Edit: They are `global_breakpoint_{xs,sm,md,lg,xl,2xl}`.

Reply:
Thanks - implemented these variables from react-tokens.